c/tskit/core.h

```diff
@@ -247,7 +247,6 @@ not found in the file.
 #define TSK_ERR_MUTATION_PARENT_EQUAL                               -501
 #define TSK_ERR_MUTATION_PARENT_AFTER_CHILD                         -502
 #define TSK_ERR_MUTATION_PARENT_INCONSISTENT                        -503
-#define TSK_ERR_INCONSISTENT_MUTATIONS                              -504
```

Comment:
should probably decrement subsequent error codes?

Reply:
I'm good either way - we definitely won't do that sort of thing after 1.0, but we're not there yet so :shrug:

Reply:
I've renumbered in https://github.com/tskit-dev/tskit/pull/1233/commits/614c001afe1d468b9c3698a2c4196e415a910202